File: C_Programming/Day_4/program22.c
//Description : Write a program that accepts number from the user & display its
//              even digits

#include<stdio.h>

void DisplayEvenDigits(int iNo )
{
    int iDigit = 0;

    while(iNo != 0)
    {
        iDigit = iNo % 10;
        if(iDigit % 2 == 0)
        {
            printf("%d\t",iDigit);
        }
        
        iNo = iNo / 10;
    }

}

int main()
{
    int iValue = 0;

    printf("Enter the number : ");
    scanf("%d",&iValue);

    DisplayEvenDigits(iValue);

    return 0;
}

File: Python_Programming/Day_4/program22.py
# Description : Write a program tha accepts number from the user & display its
#               even digits

def DisplayEven(No):
    Digit = 0

    while(No != 0):
        Digit = No % 10
        if(Digit % 2 == 0):
            print(Digit , "\t" , end = "")
        No = No // 10

def main():

    Value = 0

    print("Enter the number : ")
    Value = int(input())

    DisplayEven(Value)

if __name__ == "__main__":
    main()